src/PerfView.Tests/StackViewer/StackWindowTests.cs

```diff
@@ -277,6 +277,78 @@ public Task TestSetTimeRangeWithSpaceAsync()
             return TestSetTimeRangeWithSpaceImplAsync(CultureInfo.CurrentCulture);
         }
 
+        [WpfFact]
+        [WorkItem(2179, "https://github.com/Microsoft/perfview/issues/2179")]
+        [UseCulture("en-US")]
+        public Task TestSetTimeRangeAfterGotoCalleesAsync()
+        {
+            Func<Task<StackWindow>> setupAsync = async () =>
```

Comment:
OPINION: these would be much more readable as inline functions instead of rendered as delegates

Reply:
I'm actually not sure why these are inline at all. They don't seem to be capturing anything

Reply:
Looks like this is maybe a "when in Rome" situation :)

Reply:
Probably - there are several others that follow this pattern.